autovalet_system/src/autovalet_system.py

```diff
@@ -61,7 +68,10 @@ def __init__(self,sim):
         self.laneCloud_topic = "/lane/pointCloud"
         self.egoLine_topic   = "/lane/egoLine"
         self.color_topic     = "/frontCamera/color/image_raw"
-        self.depth_topic     = "/depth_registered/image_rect"
+        self.depth_topic     = "/depth_registered/image_rect" if self.sim else \
+                               "/frontCamera/aligned_depth_to_color/image_raw"
+        print(self.depth_topic, self.color_topic)
```

Comment:
prob dont need this

Reply:
why not? I don't recall we avoid this problem anywhere else. okay maybe @SubramanianKrish  remembers where it's solved?

Reply:
I don't remember the solution, I pointed out another place which could possibly break it

Reply:
yea I looked everywhere. I think this is the place we define what the depth topic should be, so should leave it as it is.

Reply:
lol already forgot why i commented here but i think i just meant we dont need the print statement

Reply:
lol ok hahaha

Reply:
So I understand this is the only place we define it. Thanks for looking into it Eva!